docs/content/configuration/csi_ug_config_advanced_importvol.md

```diff
@@ -1,115 +1,106 @@
 # Importing an existing volume
 
-Use this information to import volumes created externally from the IBM® block storage CSI driver by using a persistent volume (PV) yaml file.
+Use this information to import volumes created externally from the IBM® block storage CSI driver by using a persistent volume (PV) YAML file.
 
-Before starting to import an existing volume, find the following information in the existing volume in order to include the information in the persistent volume (PV) yaml file:
+Before starting to import an existing volume, find the following information in the existing volume in order to include the information in the persistent volume (PV) YAML file:
+- `volumeHandle`
+- `volumeAttributes` (optional)
+  
+  Including:
 
--   `volumeHandle`
--   `volumeAttributes` (optional)
-
-    Including:
-
-    -   `pool_name`: _Name of Pool where volume is located_ (Listed as `pool_id` for DS8000® Family systems.)
-    -   `storage_type`: <SVC | A9K | DS8K>
-    -   `volume_name`: _Volume name_
-    -   `array_address`: _Array address_
+    - `pool_name`: _Name of Pool where volume is located_ (Listed as `pool_id` for DS8000® Family systems.)
+    - `storage_type`: <`SVC` | `A9K` | `DS8K`>
```

Comment:
@oriyarde
A9K, its not A9000?

Reply:
I think you're right, not sure how `A9K` got there

Reply:
though `A9K` would've been better IMO

Reply:
if we will change it now it will be not BC..

Reply:
so, @rivkap please change `A9k` to `A9000` here

Reply:
DONE 
This is how it was given to me for [first release](https://www.ibm.com/docs/en/stg-block-csi-driver/1.0.0?topic=configuration-importing-existing-volume)